Markdown/Shell (for Markdown).sublime-syntax

```diff
@@ -8,8 +9,18 @@ extends: Packages/ShellScript/Shell-Unix-Generic.sublime-syntax
 
 contexts:
   prototype:
-    - meta_append: true
+    - meta_prepend: true
+    # continuation lines begin with `> `
+    - match: ^>(?=\s)
+      scope: comment.other.shell
+
+  main:
+    - match: ^(?=\$\s)
```

Comment:
A `#` is frequently used to signal a shell with root permissions as opposed to a `$`, e.g. throuhout the entire Arch Wiki. Can we include that as well?

Reply:
It may clash with normal shell comments.

The following example is found at https://github.com/dylanaraps/pure-bash-bible


![grafik](https://user-images.githubusercontent.com/16542113/156205830-bc8f8f77-c1c7-4f9a-8fff-58cfb6e75015.png)

The current solution treats everything as ordinary bash script until a `$` begins a line as `shell` is sometimes used as long version of `sh`. Adding `#` would cause this heuristic to fail at the first comment.